### PR TITLE
[member #2] Create Genre class

### PR DESCRIPTION
**Pull request by RaphDasilva**
# I did the following

- Created a Genre class associated with the Item class (in a music_genre.rb file).
- Added add_item method in Genre class.
- Fix linters error

### CODE REVIEW

**Review by anthonymr — CHANGES_REQUESTED** [review]
Hi @RaphDasilva, good job so far. Please, take a look at the comments below in order to complete the task.

File: bin/music_genre.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Genre
```
Comment (anthonymr) [inline bin/music_genre.rb:1]: Please, rename the file for this class from `music_genre.rb` to `genre.rb`. Remember the file name should match the class name using the right style: `snake_case` for file and `UpperCamelCase` for class.

File: bin/music_genre.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  attr_accessor :items, :name
```
Comment (anthonymr) [inline bin/music_genre.rb:2]: Please, note that in the UML diagram, the `item` attribute is private.

File: bin/music_genre.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+  def initialize(name)
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
```
Comment (anthonymr) [inline bin/music_genre.rb:10]: Good job on this method, please, don't forget to add the `add_gender` method to the `MusicAlbun` class according to the diagram.

**Review by anthonymr — CHANGES_REQUESTED** [review]
Hi @RaphDasilva, good job so far. Please, take a look at the comments below in order to complete the task.

File: bin/music_genre.rb (hunk 3 above)
Comment (RaphDasilva) [reply bin/music_genre.rb]: Thank you very much, I have done the corrections

**Review by AbdimulhinYussuf3675 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi @RaphDasilva  
Your project is complete! There is nothing else to say other than... Great job 👏🏾  Congratulations! :tada:
 
 TO HIGHLIGHT:
 

- [x]   Implemented required tasks :white_check_mark:
- [x]  Implemented all changes requested by the previous reviewer 💯 
- [x]   Good Job :+1: :smile: 
 
 Now you can merge this branch and move on 🥇 
 
 ## Optional suggestions

- [x] N/A keep doing the great work :wink: 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @AbdimulhinYussuf3675 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**